app/views/layouts/application.html.erb

```diff
@@ -16,121 +16,7 @@
-                  class: 'nav-link text-white bigger' %>
-            </li>
-          </ul>
-
-          <ul class="navbar-nav mb-md-0">
-            <% unless user_signed_in? %>
-              <li class="nav-item navbar-dark">
-                <%= link_to 'Adopt', new_user_registration_path,
-                    class: 'btn-outline-light btn me-3 navbar-margin-bottom bigger' %>
-              </li>
-              <li class="nav-item navbar-dark">
-                <%= link_to 'Log In', new_user_session_path, 
-                    class: 'btn-outline-light btn bigger' %>
-              </li>
-            <% end %>
-
-            <% if user_signed_in? %>
-              <li class="nav-item dropdown me-3 navbar-margin-bottom">
-                <a class="nav-link dropdown-toggle text-white bigger" href="#" 
-                  id="dropdown04" data-bs-toggle="dropdown" aria-expanded="false">
-                  Dashboard
-                </a>
-                <ul class="dropdown-menu" aria-labelledby="dropdown05">
-                  <% if current_user.adopter_account && 
-                        current_user.adopter_account.adopter_profile.nil? %>
-                    <li>
-                      <%= link_to "My Profile", new_profile_path, 
-                          class: 'dropdown-item bigger' %>
-                    </li>
-                  <% elsif current_user.adopter_account %>
-                    <li>
-                      <%= link_to "My Profile", profile_path, 
-                          class: 'dropdown-item bigger' %>
-                    </li>
-                    <li>
-                      <%= link_to "My Applications", my_applications_path, 
-                          class: 'dropdown-item bigger' %>
-                    </li>
-                  <% end %>
-                  <% if current_user.staff_account &&
-                        current_user.staff_account.verified %>
-                    <li>
-                      <%= link_to "Our Pets", pets_path, 
-                          class: 'dropdown-item bigger' %>
-                    </li>
-                    <li>
-                      <%= link_to "Applications", adopter_applications_path, 
-                          class: 'dropdown-item bigger' %>
-                    </li>
-                  <% end %>
-                  <li>
-                      <%= link_to "Account Settings", edit_user_registration_path, 
-                          class: 'dropdown-item bigger' %>
-                    </li>
-                </ul>
-              </li>
-              <li class="nav-item navbar-dark">
-                <%= button_to 'Log Out', destroy_user_session_path, 
-                    class: 'btn-outline-light btn bigger', 
-                    data: { turbo: false },
-                    method: 'delete' %>
-              </li>
-            <% end %>
-          </ul>
-        </div>
-      </div>
-    </nav>
-
+    <%= render "layouts/header" %>
```

Comment:
I would either pass in organization or create a helper called `current_organization` to fetch the organization to configure the page views.